database/src/main/kotlin/no/nav/su/se/bakover/database/behandling/BehandlingRepo.kt

```diff
@@ -22,6 +23,7 @@ interface BehandlingRepo {
     fun slettBeregning(behandlingId: UUID)
     fun settSaksbehandler(behandlingId: UUID, saksbehandler: NavIdentBruker.Saksbehandler)
     fun oppdaterAttestant(behandlingId: UUID, attestant: NavIdentBruker.Attestant)
+    fun oppdaterUnderkjentAttestering(behandlingId: UUID, attestant: NavIdentBruker.Attestant, underkjennelse: Attestering.Underkjennelse)
```

Comment:
Hadde det vært nok med en funksjon som het "oppdaterAttestering"? Kunne tatt inn en sealed type Attestering som kunne blitt skrevet rett til json?

Reply:
Yes, räcker med en funktion hvis Attestering hade varit en sealed type 😊 Grunnen till att det blev två funktioner nå var før att man kunde springa in i problemet att "glömma" lägga in en underkjennelsesgrunn når ting är nullable.